src/main/java/core/basesyntax/dto/StorageFruitDto.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StorageFruitDto {
+    public static final List<FruitDto> fruitAndActionList = new ArrayList<>();
```

Comment:
I thinks single storage for fruits is enough we don't need second one for dtos

Reply:
done